.gitignore

```diff
@@ -0,0 +1,2 @@
+write.json
```

Comment:
Let's remove these, we should consider them.

Reply:
Done

Reply:
Done